app/controllers/broken_links_export_request_controller.rb

```diff
@@ -0,0 +1,16 @@
+class BrokenLinksExportRequestController < ApplicationController
+  def show
+    if user_signed_in?
+      filename = "broken-link-reports-#{params[:export_id]}.zip"
+
+      begin
+        file = S3FileHandler.get_file_from_s3(filename)
+        send_data(file, filename: filename)
+      rescue Fog::AWS::Storage::NotFound
+        head :not_found
+      end
```

Comment:
We can remove the begin block: https://www.rubydoc.info/gems/rubocop/0.27.0/RuboCop/Cop/Style/RedundantBegin

I was also wondering if it would be better to use a conditional rather than an exception for flow control here. 🤔 Something like `if file... else...`.

Reply:
I think the conditional was tried in one of the other places this was used (`publisher` perhaps) — I think the slight issue is that it requires an extra call to S3 in order to do so.

But perhaps it would make sense for `S3FileHandler` to handle that exception and return `nil` if there's no file, so that we can have a conditional here at least.

Reply:
> In fact `S3FileHandler` raises the exception if `file` is `nil`.
Ah, that's a shame. Oh, well! 🤷‍♂️

Reply:
@AlanGabbianelli no, that's good :) It's our code so easily changed.